src/java/com/unifina/data/HistoricalEventQueue.java

```diff
@@ -154,6 +154,11 @@ public boolean process(FeedEvent event) {
 		return true;
 	}
 
+	@Override
+	protected EventQueueMetrics retrieveMetricsAndReset() {
+		return new EventQueueMetrics();
```

Comment:
Historical runs don't contribute to metrics? (delay metric doesn't make sense, otherwise should be applicable)